packages/dd-trace/src/llmobs/plugins/anthropic.js

```diff
@@ -0,0 +1,281 @@
+'use strict'
+
+const LLMObsPlugin = require('./base')
+
+const ALLOWED_METADATA_KEYS = new Set([
+  'max_tokens',
+  'stop_sequences',
+  'temperature',
+  'top_k',
+  'top_p',
+])
+
+class AnthropicLLMObsPlugin extends LLMObsPlugin {
+  static integration = 'anthropic' // used for llmobs telemetry
+  static id = 'anthropic'
+  static prefix = 'tracing:apm:anthropic:request'
+
+  constructor () {
+    super(...arguments)
+
+    this.addSub('apm:anthropic:request:chunk', ({ ctx, chunk, done }) => {
+      const chunks = ctx.chunks ?? (ctx.chunks = [])
```

Comment:
I had to read this line a few times and bust out a REPL to confirm understanding.

This isn't as terse but it's more understandable IMO:

```suggestion
      ctx.chunks = ctx.chunks || []
      const chunks = ctx.chunks
```

Reply:
ooh good callout, will simplify this!